language/samples/v1/samplegen_map_field_access.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "samplegen_map_field_access")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-language
+
+# sample-metadata
+#   title: This sample reads and loops over a map field in the response
+#   description: This sample reads and loops over a map field in the response
+#   usage: python3 samples/v1/samplegen_map_field_access.py
+
+# [START samplegen_map_field_access]
+from google.cloud import language_v1
+from google.cloud.language_v1 import enums
+
+
+def sample_analyze_entities():
+    """This sample reads and loops over a map field in the response"""
+
+    client = language_v1.LanguageServiceClient()
```

Comment:
Can we add a comment to clarify the role of the client? Some clients generate background resources and should only be created once and used multiple times, while others are just wrappers around HTTP calls. I'm not sure which this client falls under, but I think it would be useful to signal this to users.

Reply:
All of the generated clients are just wrappers around gRPC/HTTP calls.

Reply:
> Can we add a comment to clarify the role of the client?

Yes!

In lots of languages, there is a 1 line comment above this line. Usually simply:

```
# Initialize client
client = new library.SomeServiceClient()
```

But at least it calls attention do it!

----

TAL at [the layout Node.js uses](https://github.com/googleapis/google-cloud-node/blob/f0570e6f3d7f7fb49d5211ad7f8ed6d11809546c/generated-sample-code/samplegen_basics.js) for inspiration ([Node PR](https://github.com/googleapis/google-cloud-node/pull/2959))

> Just look at the comments, I like how it organizes/structures the sample:

```js
function main(displayName = 'This is the default value of the display_name request field') {
  // [START samplegen_basics]
  /**
  * TODO(developer): Uncomment these variables before running the sample.
  */
  // const displayName = 'This is the default value of the display_name request field';

  // Imports the client library
  const {ProductSearchClient} = require('@google-cloud/vision').v1;

  // Instantiates a client
  const productSearchClient = new ProductSearchClient();

  async function sampleCreateProductSet() {

    // Construct request
    const request = {
      parent: formattedParent,
      productSet: productSet,
    };

    // Run request
    const [response] = await productSearchClient.createProductSet(request);

    // The API response represents the created product set
    const productSet = response;
    console.log(`The full name of the created product set: ${productSet.name}`);
  }
  sampleCreateProductSet();
  // [END samplegen_basics]
}
```